insights/parsers/kdump.py

```diff
@@ -285,6 +286,9 @@ class SysconfigKdump(SysconfigOptions):
     These are set to the value of the named variable in the kdump sysconfig
     file, or '' if not found.
     """
+    def __init__(cls, *args, **kwargs):
```

Comment:
Same comment as above regarding use of `cls`.

Reply:
@PaulWay - Hi, How do you think about the comments Bob commented? And BTW, the shippable test failed, would you mind to fix it at first?